### ISSUE TITLE
evaluating on test set

### ISSUE THREAD

**Issue by ritchieng** [easonnie/multiNLI_encoder#3]
i'm receiving pytorch errors when evaluating on the test set instead of the dev set, may I know how would you recommend evaluating on the test set via `eval_model`?

trying to replicate your test results. thanks!

```
k: [0,0,0], thread: [30,0,0] Assertion `t >= 0 && t < n_classes` failed.
/opt/conda/conda-bld/pytorch_1518243271935/work/torch/lib/THCUNN/ClassNLLCriterion.cu:101: void cunn_ClassNLLCriterion_updateOutput_kernel(Dtype *, Dtype *, Dtype *, long *, Dtype *, int, int, int, int, long) [with Dtype = float, Acctype = float]: block: [0,0,0], thread: [31,0,0] Assertion `t >= 0 && t < n_classes` failed.
```

On checking, it's returning 0 correct predictions which explains this error. This doesn't happen for the validation set where it's returning the right values according to what was saved in the training phase.

**Comment by ritchieng** [comment]
@easonnie Any updates? :) Thanks!

**Comment by easonnie** [comment]
@ritchieng 
The multiNLI test set is unlabeled. You can evaluate models' performance on the test set by submitting the results on Kaggle. 

Matched:
https://www.kaggle.com/c/multinli-matched-open-evaluation/leaderboard
MisMatched:
https://www.kaggle.com/c/multinli-mismatched-open-evaluation/leaderboard
